app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -148,4 +192,15 @@ class HomeFragmentPresenter @Inject constructor(
   fun onTopicSummaryClicked(topicSummary: TopicSummary) {
     routeToTopicListener.routeToTopic(topicSummary.topicId)
   }
+
+// TODO(#555): Create one central utility file from where we should access date format or even convert date timestamp to string from that file.
+  private fun displayGreeting(){
+    val c = Calendar.getInstance()
+    when (c.get(Calendar.HOUR_OF_DAY)) {
+      in 0..11 -> userAppHistoryViewModel.greeting = fragment.requireContext().getString(R.string.good_morning)
+      in 12..15 -> userAppHistoryViewModel.greeting = fragment.requireContext().getString(R.string.good_afternoon)
+      in 16..20 -> userAppHistoryViewModel.greeting = fragment.requireContext().getString(R.string.good_evening)
+      in 21..23 -> userAppHistoryViewModel.greeting = fragment.requireContext().getString(R.string.good_night)
```

Comment:
Lets update these timings like this:

5-11 -> Morning
12 - 16 -> Afternoon
17 - 22 -> Evening
23 - 4 -> Night

Lets finish this for now, we can discuss this in meeting again and if there are changes it will act as a good started issue for a new developer.

Reply:
Okay, I have updated these timings.